lib/plugins/plugins.js

```diff
@@ -185,17 +185,21 @@ const getPublicUrl = config => {
     .join('/')
 }
 
-function getFileSystemPath (config) {
-  throwIfBadFilepath(config)
+function getFileSystemPath ({ packageName, item }) {
+  // item will either be the plugin path or will be an object containing the plugin path within the src property
+  item = item.src || item
```

Comment:
I initially suggested `src` but I'm having second thoughts.

I think this should be `path` so it's not too js specific.

Reply:
Done